package.json

```diff
@@ -90,7 +90,7 @@
     "es5-ext": "0.10.53",
     "debuglog": "1.0.1",
     "debug": "4.3.4",
-    "knex": "0.95.15",
+    "knex": "2.4.0",
```

Comment:
We also need to double check `adapterCache` with this change

Reply:
Yup, that's a great suggest. But do we have tests of the caching mechanism?

Reply:
yea